types/relay-runtime/index.d.ts

```diff
@@ -51,6 +51,15 @@ export type RequestNode = ConcreteRequest | ConcreteBatchRequest;
 // tslint:disable-next-line:no-const-enum
 export const enum FragmentReference {}
 
+export interface OperationBase {
```

Comment:
I’m not opposed to this name, but I’m open to better suggestions.

Reply:
IMO this fits the [naming](https://github.com/facebook/relay/blob/master/packages/relay-runtime/store/RelayStoreTypes.js#L275) used in Relay